README.md

```diff
@@ -47,6 +48,11 @@ $ ./plugin-cli.js
 
 #### Importing the base CLI and registering all of the base commands (`copy`, `dotnet`, `dotnet-test`, etc...)
 
+<details>
```

Comment:
Thought: Is there any way to indent the content of these given sections (ie. #### Importing the base CLI...) - right now it renders with these content areas actually outdented further than the headings.

Reply:
https://share.getcloudapp.com/L1uQ5bW6

Reply:
Hmm - I think we can indent the code within the block, but not the block itself. I did wrap these in collapsible panels so the overview of the readme is less dense. Is this better?

![image](https://user-images.githubusercontent.com/11774799/94579348-b73b1180-0246-11eb-97ed-cbcaf13c639c.png)

Reply:
Yeah, that's great 👍